# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../openmotif/Imakefile-openmotif-2.1.30.diff
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -Nur motif-vanilla/config/Imakefile motif/config/Imakefile
--- motif-vanilla/config/Imakefile	2003-11-22 15:54:21.032310256 +0100
+++ motif/config/Imakefile	2003-11-22 15:55:04.876644904 +0100
@@ -5,8 +5,7 @@
 #undef BootstrapCleanSubdirs
 #define BootstrapCleanSubdirs BOOTSTRAPSUBDIRS="$(BOOTSTRAPSUBDIRS)"
 
-BOOTSTRAPSUBDIRS = imake makedepend
-SUBDIRS = cf $(BOOTSTRAPSUBDIRS) util
+SUBDIRS = 
 
 MakeSubdirs($(SUBDIRS))
 DependSubdirs($(SUBDIRS))
